ksql-engine/src/main/java/io/confluent/ksql/function/UdfCompiler.java

```diff
@@ -101,6 +104,7 @@ UdfInvoker compile(final Method method, final ClassLoader loader) {
    *<pre>{@code
    * // try and coerce the arguments to the types expected by the UDF
    * Boolean arg0;
+   * if (args[0] == null && isPrimitive) throw new KsqlFunctionException(...);
```

Comment:
I think, given the new more-readable `genericTemplate` you can probably remove this comment. It will only get out of sync.